doc/lsst.meas.algorithms/creating-a-reference-catalog.rst

```diff
@@ -155,3 +158,40 @@ You can query the ``refcats`` collection to see whether your htm shards appear:
     butler query-datasets --collections refcats REPO
 
 For LSST staff using ``lsst-devl``, see the `Reference catalogs policy <https://developer.lsst.io/services/datasets.html#reference-catalogs>`_ in the Developer Guide for additional policy about adding reference catalogs to the common repo.
+
+.. _lsst.meas.algorithms-refcat-existing:
+
+5. What to do with existing reference catalogs?
+===============================================
+
+Already existing reference catalogs (for example, the PS1 or Gaia DR2 catalogs that were used in gen2 butlers) can also be ingested into the butler, negating the need to convert them to the LSST format yourself.
+To ingest an already existing converted reference catalog, first create a suitable filename to htm7-index astropy lookup table, and then follow the steps above to `ingest the files into the butler <lsst.meas.algorithms-refcat-ingest>`.
+
+This is an example script that constructs a conversion lookup table for all converted FITS files in the `refcat_dir` directory, outputting to the `out_dir` directory:
```

Comment:
"that constructs..." -> ```that creates an ``.ecsv`` file for the ``butler ingest-files`` command, from all of the HTM indexed files in a given directory. We use the existing Gaia DR2 catalog on lsst-devl in this example:```

Reply:
`This is an example script that creates an ``.ecsv`` lookup table for the ``butler ingest-files`` command, from all of the HTM indexed files in a given directory (`refcat_dir` here). We use the existing Gaia DR2 catalog on lsst-devl in this example:`